Question paper

```diff
@@ -39,3 +39,108 @@
 2. How will you synchronise 3 threads by using 3 semaphores ---5 mins
 3. Explain the difference between malloc and calloc ---5 mins
 
+
+
+
+ANSWER
+SECTION 1:
+
+2. WAP to find the prme number------------------------5 mins
+ANSWER:
+   int no;
+   printf("enter a number:\n");
+   scanf("%d",&no);
+   if((no%2)!=0&&(no%3)!=0){
+      printf(" %d is prme number\n",no);
+   }
+   else{
+      printf("%d is not prme number");
+   }
+   
+   SECTION 3:
+   
+   1. What is the difference between run time and load time libraries-----5 mins
+      1.run time : we link libraries at excution time
+        load time : we link libraries at load time
+      2.run time : libraries prefer when we want only in perticular area in program. 
+        load time : labraries prefer when we want whole program.
+      
+      
+   3. What is the difference between detachable and joinable thread-----5 mins
+      joinable thread are wait till main process can terminate,where detachable thread are not depend on main thread after the completion they ill be terminate.
```

Comment:
The statement is technically correct but looks like the statement is incomplete

Reply:
Just improving the grammer and technicality: "THe resources of the joinable threads are released by the main thread, so for that apon exiting the exit status is send to the main thread, But detachable threads do not send the exit status to main thread "